specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -456,32 +534,44 @@ namespace Microsoft.Windows.PushNotifications
+        static void UnregisterActivator(PushNotificationRegistrationToken token, PushNotificationRegistrationOptions options);
 
         // Request a Push Channel with an encoded RemoteId from WNS. RemoteId is an AAD identifier GUID
         static Windows.Foundation.IAsyncOperationWithProgress<PushNotificationCreateChannelResult, PushNotificationCreateChannelStatus> CreateChannelAsync(Guid remoteId);
+
+        // Applications will call this to check which flags are supported for RegisterActivator
+        static Boolean IsActivatorSupported(PushNotificationRegistrationOptions options);
     };
 }
+
+
+
 ```
+
 # Appendix
-* For all OS images before 21H1 (19043), the RemoteId will be a noop and the native platform will </br>
-send the PFN (PackageFamilyName) in the channel request protocol. The WNS server will maintain an </br>
-internal mapping of PFN -> RemoteId and will return a channelUri with the encoded RemoteId after querying </br>
-the map.
-* For OS images 21H1(19043) and beyond, the Windows App SDK component will call into newly added Closed source </br>
-APIs in the OS to actually pass a RemoteId in the Channel Request operation.
-* For unpackaged Win32 apps, the Windows App SDK component will call into Closed source APIs to register the </br>
-unpackaged Win32 process with the RemoteId.
-* A long running Windows App SDK component will be needed to intercept Push payloads from the native platform and </br>
-Launch the corresponding unpackaged Win32 app. We will be using Protocol Activation via WinMain to launch the </br>
-unpackaged Win32 process from the long running service. The requirement is mainly because the native client </br>
-today does not have support to launch Win32 apps directly in response to a Push payload unless they are </br>
-already running in which case we simply marshall the payload back to the process via a registered callback sink.
-* For packaged applications like MSIX Desktop Bridge, we buid a thin wrapper around existing Background Manager </br>
-APIs rather than re-inventing new Background Triggers. This is mainly because Background Infrastructure native </br>
-client stack is built with Power Management and resourcing concerns already addressed. Building a new stack that </br>
-addresses similar concerns is likely not the best use of our time or resources. Instead we should invest in </br>
-Windows App SDK-izing the Background Infra stack itself.
+
+-   For all OS images before 21H1 (19043), the RemoteId will be a noop and the native platform will
+    send the PFN (PackageFamilyName) in the channel request protocol. The WNS server will maintain
+    an internal mapping of PFN -> RemoteId and will return a channelUri with the encoded RemoteId
+    after querying the map.
+-   For OS images 21H1(19043) and beyond, the Windows App SDK component will call into newly added
+    Closed source APIs in the OS to actually pass a RemoteId in the Channel Request operation.
```

Comment:
Don't need to call out its open/closed state. For the purposes of this spec, it's an OS provided API. Anything beyond that is out of scope.

Same suggestion applies to other instances below.

```suggestion
    APIs in the OS to pass a RemoteId in the Channel Request operation.
```